test/functional/filter_test.rb

```diff
@@ -21,7 +24,7 @@ def assert_update_success(result)
 
   describe 'create' do
```

Comment:
Thanks, @yifatmakias! +1 for using default rubocop :rose:.

Could you please also add a test for `create` action which actually tries to create a filter? The one we have there uses `api_expects_no_call` and misses a pair.

Reply:
Yes, I will add another test for create :)